yt_dlp/extractor/crunchyroll.py

```diff
@@ -412,6 +422,12 @@ def _real_extract(self, url):
         webpage = self._download_webpage(
             self._add_skip_wall(webpage_url), video_id,
             headers=self.geo_verification_headers())
+        if re.search(r'<div id="preload-data">', webpage):
+            initial_state, app_config = self._get_embedded_json(webpage, video_id)
+            base_site = app_config['baseSiteUrl']
+            path = initial_state['router']['locations']['current']['pathname']
+            self.to_screen(f'{video_id}: Redirected to beta site. Switching extractors. New url: {base_site}{path}')
```

Comment:
Verbose log will automatically show this
```suggestion
```

Reply:
I actually feel that this message should be there even without the verbose flag. There are some differences in what the result file looks like from the beta vs non-beta extractors (different video ids, for example), and this will be less confusing for users if there's a line in the normal output clearly stating what's going on.

Reply:
The following messages will show the name of the new extractor even without verbose. So I think this is unnecessary. But I don't feel strongly enough about it to force the change.